program/c/makefile

```diff
@@ -16,7 +16,7 @@ endif
 # The all target is defined by the solana makefile included above and generates the needed .o file.
 .PHONY: cpyth-bpf
 cpyth-bpf: all
-	bash -c "ar rc $(OUT_DIR)/libcpyth-bpf.a $(OUT_DIR)/oracle/*.o"
+	bash -c "ar rcs $(OUT_DIR)/libcpyth-bpf.a $(OUT_DIR)/oracle/*.o"
```

Comment:
turns out this `s` is important for making the binary size small. (also it didn't fix the ELF loading error when i tried it 🤷 )